pkg/service/utgen/assets/test_generation.toml

```diff
@@ -128,14 +212,15 @@ new_tests:
 {%- endif %}
   test_code: |
 {%- if language in ["python"] %}
+    import pytest
+    @pytest.mark.asyncio
     def ...
 {%- else %}
     ...
 {%- endif %}
   new_imports_code: |
     {%- if language == "python" %}
     "import pytest"
-    "from my_module import my_function"
     {%- elif language == "java" %}
     "import org.junit.jupiter.api.Test;"
```

Comment:
any reason for removing this? i put it there so that it can get idea of both type of import formats

Reply:
Thanks, wasnt able to find one issue just now, i guess this will solve it